### PR TITLE
Removed dead link

### PR DESCRIPTION
**Pull request by vondk**
The link does not work anymore.

## Proposed change

The link is dead, thus I would suggest to remove it. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by TomBrien** [comment]
Looks like the original link was badly formatted

File: source/hassio/installation.markdown [hunk 1]
```diff
@@ -15,8 +15,7 @@ The following will take you through the steps required to install Home Assistant
      - [Raspberry Pi 4 Model B (1 GB, 2 GB, 4 GB and 8 GB model) 64-bit][pi4-64] (64-bit is required for 8 GB model)
      - [Tinkerboard][tinker]
      - [Odroid-C2][odroid-c2], [Odroid-C4 (Beta)][odroid-c4], [Odroid-N2][odroid-n2], [Odroid-XU4][odroid-xu4]
-       - [Guide: Flashing Odroid-N2 using OTG-USB][otg-usb]
-     - [Intel NUC][intel-nuc]
+      - [Intel NUC][intel-nuc]
```
Comment (TomBrien) [inline source/hassio/installation.markdown:18]:
```suggestion
       - [Guide: Flashing Odroid-N2 using OTG-USB][otg-usb]
     - [Intel NUC][intel-nuc]
```

Comment (TomBrien) [reply source/hassio/installation.markdown:18]: Let's keep the link and change the very last line to:
```markdown
../flashing_n2_otg
```

**Review by TomBrien — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
The link has been fixed.

**Pull request closed without merging**